mr/src/test/java/org/apache/iceberg/mr/TestIcebergInputFormats.java

```diff
@@ -408,6 +408,11 @@ public String name() {
         public TestInputFormat<T> create(Configuration conf) {
           return function.apply(conf);
         }
+
+        @Override
+        public String toString() {
+          return String.format("Test%s<T>", name());
```

Comment:
@rdblue should I remove this one and instead grab it in a follow up PR? Its just part of the test suites, but the template parameter T could be grabbed via java reflection - and it is different in some of the tests.

Reply:
I'll leave the `toString` method and then open an issue for getting the proper name of the template parameter, given that we test with multiple types substituted for `T`. It can be obtained via java reflection and since this is limited to the test files, there's no real overhead outside of potentially test time (but this isn't called so often as to be a concern).

Reply:
Ok. I've decided to leave it in as stands and I've created an issue to see if I can follow up on it so that we have the different types used in the parameter groups properly displayed in the tests: https://github.com/apache/iceberg/issues/1542